### DISCUSSION TITLE
Count at refiner level is not matching the results returned initially

### DISCUSSION THREAD

**Discussion by roberAlb** [microsoft-search/pnp-modern-search#3222] [Q&A] [answered]
**Version used**
4.9.3

**Describe the bug**
We have the refiners webpart with the show count enabled per refiner values. With big values (more than 100) is displaying a number and when we select it the results are not matching with the total count. I know that the total count is something that can be different sometimes, but what about the refiners count? Could it be a bug? 

**To Reproduce**
We select the filter with 130 results and we have 150 in the total results count, we filter
![image](https://github.com/microsoft-search/pnp-modern-search/assets/50910751/7603fd7c-ed33-4b92-91ae-97509765f601)

After that, the value in the filter is displaying now 122 which is matching with the total results count which is different that was displayed at the beginning. 

![image](https://github.com/microsoft-search/pnp-modern-search/assets/50910751/f7954285-3a01-4810-8a81-0d194787b5b5)

**Expected behavior**
I would expect when I load the page that the filter has 122 results initially without applying any search

**Desktop (please complete the following information):**
 - Browser chrome, edge

**Additional context**
As I said, this is not happening with few results like 15-20-8, in these cases is displaying the right refiner count at the beginning.

**Comment by kasperbolarsen** [comment, marked as answer]
Please see https://github.com/microsoft-search/pnp-modern-search/issues/2527

The refiner counts are estimates provided by the API :-(
